framework/source/class/qx/data/Array.js

```diff
@@ -647,10 +647,22 @@ qx.Class.define("qx.data.Array",
     /**
      * Check if the given item is in the current array.
      *
+     * @deprecated {6.0} Please use the include method instead
+     *
      * @param item {var} The item which is possibly in the array.
      * @return {Boolean} true, if the array contains the given item.
      */
     contains: function(item) {
```

Comment:
this is often done in `defer`, patching instead of additional function call eg:
```
   defer : function(statics, members) {
     members.contains = members.includes;
```

Reply:
The problem with doing it in defer is that the `@deprecated` tag can't be added easily, I think

Reply:
I was going to suggest the same thing, but if we do plan to deprecate it (which I'm not convinced we should, since it's existed since the dawn of time), we would want to mark it that way.

Reply:
it can, what's been used elsewhere is to declare the aliasing method (i.e. `contains`) as just null in `members`, so it can have documentation and jsdoc

Reply:
@johnspackman I've moved that to *defer*. Lets see if the tests work out...

Reply:
Doesn't work as expected. I've reverted it. Feel free to change the PR to use *defer*. Although this doesn't save anything, because the defer block needs to be created anyway.